mono/mini/mini-exceptions.c

```diff
@@ -2508,9 +2505,9 @@ mono_print_thread_dump_internal (void *sigctx, MonoContext *start_ctx)
 		g_string_append (text, "\n\"<unnamed thread>\"");
 
 #ifndef HOST_WIN32
```

Comment:
now that you have a stub mono_threads_platform_describe on win32, does it make sense to remove this #ifndef ?